config/environments/test.rb

```diff
@@ -37,6 +37,8 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  config.action_mailer.default_url_options = { host: ENV['DEFAULT_URL_OPTION'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.